templates/PlayFab_Api.cpp.ejs

```diff
@@ -118,3 +118,5 @@ namespace PlayFab
 }
 
 #endif
+
+#pragma warning (enable: 4100) // formal parameters are part of a public interface
```

Comment:
We usually put this line at the top of files

Reply:
It is.
The line at the top disables it, the line at the bottom re-enables it.